exercises/algorithms/src/main/java/com/cbfacademy/search/Search.java

```diff
@@ -0,0 +1,17 @@
+package com.cbfacademy.search;
+
+public interface Search {
+    /**
+     * Searches an array.
+     *
+     * @param sortedArray The array to be searched (the array should already be s
+     *                    rted)
+     * @param target      The value to be searched for in the array
+     */
+    int search(int[] sortedArray, int target);
+
+    /**
+     * Returns the exercise name
+     */
+    String getName();
```

Comment:
Heya, this follows on from my mention this week that it might be worth us thinking about standardising our exercise files (e.g. comments to describe a method, etc). We have previous exercises with a getName method and it's included here for consistency, though can be removed.

Reply:
Yes, I've seen that in other exercises, but never understood the intended benefit for the students or assessors.

Very much in favour of standardisation, but let's aim for useful standards. 🙂

Reply:
I actually thought that the getName() on a few of the other tests was something you'd added. 

I see your point and happy to remove.